### PR TITLE
Add Default payment billing form button

### PR DESCRIPTION
**Pull request by ghost**
## Proposed changes
 
[Jira ticket](billingDetailsInputView)
This is only for default billing form button without summary

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

Screenshot 2022-06-15 at 5 23 15 pm

### CODE REVIEW

File: Tests/UI/CardViewControllerTests.swift [hunk 1]
```diff
@@ -33,6 +33,7 @@ class CardViewControllerTests: XCTestCase {
         let cardViewController = CardViewController()
         XCTAssertEqual(cardViewController.cardHolderNameState, .required)
         XCTAssertEqual(cardViewController.billingDetailsState, .required)
+        cardViewController.isNewUI = false
```
Comment (deepesh-vasthimal-cko) [inline Tests/UI/CardViewControllerTests.swift:36]: Can you add a test when `cardViewController.isNewUI = true `as well ?

Comment (ehab-al-cko) [reply Tests/UI/CardViewControllerTests.swift:36]: It's removed for the other PR 
https://github.com/checkout/frames-ios/pull/179

File: Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+public struct PaymentFormFactory {
+
+    public static func getBillingFormButtonView(style: CellButtonStyle, delegate: SelectionButtonViewDelegate?) -> (view: UIView, delegate: SelectionButtonViewDelegate?)  {
+        let view = SelectionButtonView()
+        view.delegate = delegate
+        view.update(style: style)
+        return (view, view.delegate)
```
Comment (alex-ioja-yang-cko) [inline Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift:9]: We may have discussed this previously, but why is there a delegate being returned?

File: Source/UI/NewUI/BillingForm/Default Implementation/Payment/BillingFormDetails/DefaultPaymentBillFormDetailsButtonCellStyle.swift [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+public struct DefaultPaymentBillingFormButtonCellStyle: CellButtonStyle {
+    public var backgroundColor: UIColor = .clear
+    public var button: ElementButtonStyle = DefaultPaymentBillingFormButtonStyle()
+    public var isMandatory: Bool = true
+    public var title: ElementStyle? = DefaultTitleLabelStyle(text: "billingAddressTitle".localized(forClass: CheckoutTheme.self))
+    public var hint: ElementStyle? = DefaultHintInputLabelStyle(text: "We need this information as an additional security measure to verify this card.")
+    public var error: ElementErrorViewStyle?
+}
+
+public class DefaultPaymentBillingFormButtonStyle: ElementButtonStyle {
```
[the remark this answers is not in the document]
Comment (ehab-al-cko) [reply Source/UI/NewUI/BillingForm/Default Implementation/Payment/BillingFormDetails/DefaultPaymentBillFormDetailsButtonCellStyle.swift]: Updated

[the remark this answers is not in the document]
Comment (ehab-al-cko) [reply Source/UI/NewUI/BillingForm/Default Implementation/Payment/BillingFormDetails/DefaultPaymentBillFormDetailsButtonCellStyle.swift]: Updated

**Review by alex-ioja-yang-cko — APPROVED** [review]
I think its just a text left that I'm not sure whether we need localised or not

**Review by deepesh-vasthimal-cko — APPROVED** [review]

**Pull request merged**